articles/azure-monitor/containers/container-insights-enable-arc-enabled-clusters.md

```diff
@@ -114,10 +114,29 @@ az k8s-extension create --name azuremonitor-containers --cluster-name <cluster-n
 
 To use [managed identity authentication](container-insights-onboard.md#authentication), add the `configuration-settings` parameter as in the following:
 
+>[!NOTE]
+>Managed identity authentication is not supported for Arc k8s connected clusters with **ARO**.
+>
+
 ```azurecli
 az k8s-extension create --name azuremonitor-containers --cluster-name <cluster-name> --resource-group <resource-group> --cluster-type connectedClusters --extension-type Microsoft.AzureMonitor.Containers --configuration-settings amalogs.useAADAuth=true
```

Comment:
change '--configuration-settings amalogs.useAADAuth=true' to '--configuration-settings amalogs.useAADAuth=false'

Reply:
Changed.